tools/FakeItEasy.Deploy/Program.cs

```diff
@@ -63,9 +64,29 @@ public static async Task Main(string[] args)
                 await UploadPackageToNuGetAsync(file, nugetServerUrl, nugetApiKey);
             }
 
+            var issueNumbersInCurrentRelease = ReleaseHelpers.GetIssueNumbersReferencedFromReleases(new[] { release });

```

Comment:
Maybe consider `using static ReleaseHelpers`? If you prefer to keep it explicit, it's fine by me, but I just wanted to mention the possibility.

Reply:
Done!